### PR TITLE
Title: Smooth Tick Option and Explore Button Animation

### LINKED ISSUE #390: [Feature Request]:  Smooth  ✅  Option  loading  and  Explore Button animation.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Suggested modifications include:

Smooth Tick Options loading : A dedicated container for tick options that ensures each tick appears smoothly one by one without affecting the position of other elements on the page.

Explore Button Animation: An explore button with a dynamic animation to maintain visibility and encourage user interaction.
The tick options should appear sequentially with smooth animations, ensuring they do not overlap and remain within their container

### Use Case

This feature enhances user experience by highlighting benefits or features one by one and making navigation easier with an animated explore button.

### Benefits

Improved User Experience: Smooth animations and an animated explore button create a visually appealing and seamless interaction.

Higher Engagement: An animated explore button captures user attention and encourages interaction.

Visual Appeal: Dynamic elements keep users engaged and interested.

### Add ScreenShots

before--

https://github.com/VaibhavArora314/StyleShare/assets/133522303/155c7029-680c-41af-a283-0f10c85394e9

after--

https://github.com/VaibhavArora314/StyleShare/assets/133522303/8fea9c22-66e8-485e-b21b-38b29d741d0e

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Prabhatyadav60**
# Pull Request

### Title
Smooth Tick Option and Explore Button Animation

### Description
Smooth Tick Options Loading: Added a container for tick options to ensure they appear smoothly one by one without affecting the position of other elements.
Explore Button Animation: Implemented an eye-catching animation for the explore button to maintain visibility and encourage user interaction.

### Related Issues
Fixes #390 

### Changes Made
Added CSS for smooth tick option animation.
Implemented animation for the explore button to enhance user engagement.

### Checklist 

- [x] I have tested the changes locally
- [ ] Documentation has been updated (if necessary)
- [x] Changes are backward-compatible

### Screenshots (if applicable)

https://github.com/VaibhavArora314/StyleShare/assets/133522303/8fe1e35b-ddfe-40d4-ae26-2d96b776a543

### Additional Notes

Footer

### CODE REVIEW

**Review by VaibhavArora314 — CHANGES_REQUESTED** [review]
looks good, but can you make this animation a little faster as it is very slow right now

**Review by VaibhavArora314 — APPROVED** [review]

**Review by Ultimateutkarsh11 — APPROVED** [review]

File: frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -12,45 +12,99 @@ import FAQ from "../components/FAQ";
...
 
 function Home() {
   console.log("Home page rerendered");
   const { t } = useTranslation();
 
-  document.title='Style Share | Welcome 🙏'
-  
+  document.title = 'Style Share | Welcome 🙏';
+
+  // Define the type for visibleItems as an array of strings
```
Comment (akbatra567) [inline frontend/src/pages/Home.tsx:24]: Could we remove this comment ? Code is readable so this is unnecessary here.

Comment (Prabhatyadav60) [reply frontend/src/pages/Home.tsx]: Sure sir ,Done!

**Review by akbatra567 — APPROVED** [review]

**Comment by Prabhatyadav60** [comment]
Sir , kindly assign gssoc label and level. Thank you !

**Pull request merged**